tchannel/peer_heap.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from .container import heap
+from .container.heap import HeapOperation
+
+
+class PeerHeap(HeapOperation):
+    def __init__(self):
+        self.peers = []
+
+    def cmp(self, i, j):
+        return self.peers[i] < self.peers[j]
```

Comment:
python provides this feature. why not use it. I don't see any strong reason I can't use it for peer